### DISCUSSION TITLE
Built In Reports

### DISCUSSION THREAD

**Discussion by Avahey** [dataplat/DataSaturdays#158] [Ideas]
One feature I used a lot while planning my event was the reports. I pulled the sessions submitted report to build the rough draft of what I wanted the schedule to look like. Some of the other ones were nice, but just being able to export the sessions submitted would be useful. 

Session Submitted Columns:

Last Name
First Name
Email Address
Session Title
Session Description
Track
Category
Level
Approved
Scheduled
Session Time Preference
Additional Notes
Shirt Size
Speaker Bio
Speaker URL
Twitter
LinkedIn URL
Blog URL
MVP
MCM
MS
Phone Number
Country
Region
City
Address1
Address2

**Comment by SQLDBAWithABeard** [comment]
If we make use of Sessionize, they have excellent Speaker management tooling. Maybe guidance on how to use this could be included in a wiki or something
